### PR TITLE
feat(AP-3811): Switch between active combinations

### PR DESCRIPTION
**Pull request by joshua-barnard-evolv**
1. You can now see and switch between active combinations for any active project. The extension fetches the environment config to see the active combinations.
2. Projects are now sorted by confirmations, then name.
3. If you have selected a combination, that project appears at the top of the list.
4. A "clear selection" link appears if you select a different combination than the one allocated. You become excluded from all other projects until you click the link, which wipes the session storage item `evolv:previewCid`.
5. Added a `background.js` service worker as per the developer docs and updated the manifest to prevent CORS issues accessing the environment config.
6. Appended "(C)" to the control combinations.

image

https://github.com/evolv-ai/chrome-extension/assets/78709720/b454c4e8-4e6c-4360-8379-53f00a3f6d55

### CODE REVIEW

File: popup.js [hunk 1]
```diff
@@ -166,6 +234,22 @@ const setAllocationsAndConfirmations = () => {
   }
 };
 
+const getCombinationLabel = (candidates, ordinal) => {
+  const isControl = ordinal === candidates[0].ordinal;
```
Comment (robertsevernsentient) [inline popup.js:239]: This assumes the candidate that is firs tin the list is control. I'm not sure we can guarantee that. We do know it is the combo with the lowest ordinal

File: manifest.json [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
-  "version": "1.0",
+  "version": "1.2",
   "manifest_version": 3,
   "host_permissions": [
-
+    "https://participants.evolv.ai/*",
+    "https://participants-stg.evolv.ai/*"
   ],
-  "permissions": [],
+  "permissions": [
```
Comment (robertsevernsentient) [inline manifest.json]: test removing these

**Review by robertsevernsentient — APPROVED** [review]

File: manifest.json (hunk 2 above)
Comment (joshua-barnard-evolv) [reply manifest.json]: Looks like it doesn't need these anymore. They were required for a different implementation I tried.

File: popup.js (hunk 1 above)
Comment (joshua-barnard-evolv) [reply popup.js:239]: I've added sort.

**Comment by joshua-barnard-evolv** [comment]
I've addressed all of the PR feedback and modified the layout and styling for how we show combinations and Control.

image

**Review by robertsevernsentient — APPROVED** [review]

**Pull request merged**